src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -4638,4 +4638,10 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_ConstantStringTooLong" xml:space="preserve">
     <value>Length of String constant exceeds current memory limit.  Try splitting the string into multiple constants.</value>
   </data>
+  <data name="ERR_LoadDirectiveOnlyAllowedInScripts" xml:space="preserve">
+    <value>#load is only allowed in scripts</value>
+  </data>
+  <data name="ERR_PPLoadFollowsToken" xml:space="preserve">
+    <value>Cannot use #load after first token in file</value>
```

Comment:
extern aliases say:  ```CS0439: An extern alias declaration must precede all other elements defined in the namespace```

The language I'm using for ```#load``` is similar to ```#r``` (we could change them both if we can think of something better).

```#!``` will say "Preprocessor directive expected" (perhaps not ideal, but in the interest of not complicating lexing any more than necessary, it's not recognized unless it's at the start of the file...not even preceding whitespace is allowed).